configuration/scripts/options/set_nml.histall

```diff
@@ -1,5 +1,5 @@
     histfreq       = 'm','d','1','h','x'
-    histfreq_n     =  1,2,6,4,5
```

Comment:
ah, you came to the same conclusion :) (I'm reviewing commit by commit). We could drop this commit and edit the previous one to also drop that modification to set_nml.histall, it would make for a cleaner commit message :)